scripts/zones/Phanauet_Channel/DefaultActions.lua

```diff
@@ -0,0 +1,6 @@
+local ID = zones[xi.zone.PHANAUET_CHANNEL]
+
+return {
+    ['Eunirange'] = { messageSpecial = ID.text.ARE_WE_THERE_YET },
```

Comment:
These lines of text should be spoken by the NPC but I couldn't figure out what event to use.

Currently looks like
![image](https://github.com/LandSandBoat/server/assets/166072302/2ba9dfcd-9605-4617-a450-24e7843c08e1)

But should look like
![image](https://github.com/LandSandBoat/server/assets/166072302/0ef738b4-8287-4b66-a5ef-761fac7c58e2)

As well as Riche:
![image](https://github.com/LandSandBoat/server/assets/166072302/607b71f1-835d-471c-bb8a-770719ff10ce)

![image](https://github.com/LandSandBoat/server/assets/166072302/d1663a29-4723-4e97-ab33-22d10ad3a3bc)

Reply:
If its an event it would be in the capture.

Reply:
![image](https://github.com/LandSandBoat/server/assets/166072302/b958a7c2-024b-4235-a097-89c192f163da)

Interesting it doesn't seem to have an event, it's just called npcChat and provides the text ID.

Reply:
Woops that was the other tonberry but it's the same regardless:

![image](https://github.com/LandSandBoat/server/assets/166072302/ff71d28a-7c56-4276-bdaa-6de2af60ae89)

Reply:
`npc:showText(7217)` or something similar. Using the variable instead of a magic number of course

Reply:
Sorry based on [this](https://github.com/LandSandBoat/server/blob/0a4fbb6117291db0ff4dfbd4f6183b83dd8ac002/src/map/lua/lua_baseentity.h#L54) you probably want 

`player:showText(npc, 7217, timeLeft)` etc

Reply:
That does work, it turns them towards you and they get stuck in that direction but it's better than nothing I suppose.